terraform/crates-io-ops-bot/deployment.tf

```diff
@@ -0,0 +1,105 @@
+// Deploys the bot to the production ECS cluster
+
+// The ECR repository is used by CI to store the container image, which will
+// then be fetched by the ECS service.
+module "ecr" {
+  source = "../shared/modules/ecr-repo"
+  name   = "crates-io-ops-bot"
+}
+
+// This just loads the data for each of these secrets
+// The values are added to ASM manually
+data "aws_ssm_parameter" "crates_io_ops_bot" {
+  for_each = toset([
+    "discord_token",
+    "heroku_api_key",
+    "build_check_interval",
+    "github_org",
+    "github_repo",
+    "github_token"
+  ])
+  name = "/prod/ecs/crates_io_ops_bot/${each.value}"
+}
+
+module "ecs_task" {
+  source = "../shared/modules/ecs-task"
+
+  name               = "crates_io_ops"
```

Comment:
Let's call it `crates-io-ops-bot` to be consistent with the rest of the task definitions (it should have the same name as the ECR repository).